DigraphyApi/Dto/TitleDto.cs

```diff
@@ -0,0 +1,16 @@
+namespace DigraphyApi.Dto;
+
+public class TitleDto
+{
+    public int Id { get; set; }
+}
+
+public class CreateTitleDto
+{
+    public string Name { get; set; }
+    public string SmallPosterUrl { get; set; }
+    public string LargePosterUrl { get; set; }
+    public DateTime ReleasedAtUrc { get; set; }
```

Comment:
```suggestion
    public DateTime ReleasedAtUtc { get; set; }
```

I imagine it'll mess with the automatic mapping, but I'll get it fixed in my branch shortly I promise! 😅